lib/experimental/fonts-library/class-wp-font-family-utils.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Fonts Family Utils class.
+ *
+ * This file contains utils fot Font Family class.
+ *
+ * @package    Gutenberg
+ * @subpackage Fonts Library
+ * @since      X.X.X
+ */
+
```

Comment:
See comment here https://github.com/WordPress/gutenberg/pull/52704/files#r1269629926 for adding a guard to avoid loading this class into memory after it is introduced into WordPress Core.

Reply:
Done!